pkg/vdr/panacea_vdr.go

```diff
@@ -44,6 +45,35 @@ func (r *PanaceaVDR) Resolve(didID string, _ ...vdr.DIDMethodOption) (*did.DocRe
 		return nil, fmt.Errorf("failed to parse DID document: %w", err)
 	}
 
+	var vms []did.VerificationMethod
+	for _, vm := range doc.VerificationMethod {
+		if btcec.IsCompressedPubKey(vm.Value) {
+			pubKey, err := btcec.ParsePubKey(vm.Value, btcec.S256())
+			if err != nil {
+				return nil, fmt.Errorf("invalid secp256k1 public key of verification method: %w", err)
+			}
+
+			vm.Value = pubKey.SerializeUncompressed()
+		}
+		vms = append(vms, vm)
+	}
+
+	doc.VerificationMethod = vms
+
+	var auths []did.Verification
+	for _, auth := range doc.Authentication {
+		if btcec.IsCompressedPubKey(auth.VerificationMethod.Value) {
+			pubKey, err := btcec.ParsePubKey(auth.VerificationMethod.Value, btcec.S256())
+			if err != nil {
+				return nil, fmt.Errorf("invalid secp256k1 public key of authentication: %w", err)
+			}
+			auth.VerificationMethod.Value = pubKey.SerializeUncompressed()
+		}
+		auths = append(auths, auth)
+	}
+
+	doc.Authentication = auths
```

Comment:
I also added this code block as temporal workaround (test completed).
When authentication exists (like panacea DID doc), public key fetcher fetches the public key from authentication.
I figured out how it works while debugging, but I haven't found the right solution yet.
I'll look into it a bit more and if there's a better way, I'll share.